src/test/test_bitcoin.h

```diff
@@ -45,6 +45,12 @@ struct BasicTestingSetup {
 
     explicit BasicTestingSetup(const std::string& chainName = CBaseChainParams::MAIN);
     ~BasicTestingSetup();
+
+    fs::path getPath(const std::string& name);
+
+    private:
+        fs::path pathRoot;
+        bool cleanupRoot;
```

Comment:
nit: New members should follow the naming guidelines:

```cpp
m_path_root;
m_cleanup_root;

Reply:
Also, the members should probably be const and initialized in the constructor initializer list?